### PR TITLE
Add save and delete shortcuts

### LINKED ISSUE #7975: Keyboard Shortcuts Not Working on Certain Pages
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

Some pages support the "save" keyboard shortcuts, while others do not. We need to ensure that all forms in the App support this... otherwise you get randomly prompted by the browser to save the HTML page. 😐 

### To Reproduce

Hit `CMD-S` (Mac) on Project Settings

### What version of Directus are you using?

RC-92

### What version of Node.js are you using?

Node: 16.6.1

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying Directus?

DO

### PR DESCRIPTION
**Pull request by Nitwel**
closes #7975
Let me know if Safari has a problem with the `delete` shortcut as I only could test FF, Chrome and Edge.

### CODE REVIEW

File: app/src/modules/collections/routes/collection.vue [hunk 1]
```diff
@@ -365,6 +366,10 @@ export default defineComponent({
 
 		const { batchEditAllowed, batchArchiveAllowed, batchDeleteAllowed, createAllowed } = usePermissions();
 
+		useShortcut('delete', () => {
+			if (selection.value.length > 0) confirmDelete.value = true;
+		});
```
Comment (rijkvanzanten) [inline app/src/modules/collections/routes/collection.vue:371]: On macOS, it's more common to have this be CMD+Backspace 🤔

Comment (rijkvanzanten) [reply app/src/modules/collections/routes/collection.vue:371]: Seems like Ctrl+D would be more appropriate on windows

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: app/src/modules/collections/routes/collection.vue (hunk 1 above)
Comment (Nitwel) [reply app/src/modules/collections/routes/collection.vue:371]: Haha, didn't even knew Ctrl+D is a thing. 😄

Comment (rijkvanzanten) [reply app/src/modules/collections/routes/collection.vue:371]: Can we / should we do Windows / Mac specific keyboard shortcuts 🤔 

Ctrl+D wouldn't make any sense on a mac (as ctrl isn't used for anything like that)

Comment (Nitwel) [reply app/src/modules/collections/routes/collection.vue:371]: I don't think so and I thought that `delete` would be a standard keyboard shortcut across all major platforms.

Comment (rijkvanzanten) [reply app/src/modules/collections/routes/collection.vue:371]: macOS would use backspace, not delete 🥲

**Comment by benhaynes** [comment]
Rijk wants to use this PR as a single place for discussing all keyboard shortcuts in the App. So here goes! All of these are up for discussion, they are just ideas... and some may need to be pulled into dedicated PRs/issues.

* `META-ENTER` — Submit comments (and modals?)
* `META-A` — Select all items in current Layout
* `META-S` — Save and stay (overrides browser save)
* `META-W` — Use browser close, but confirm before leaving if unsaved changes
    * I think we already have this
* `ARROWS` — Navigate items on the layout, left/right can be used for _nested_ lists
* `SPACE` — Select the current item in a Layout
* `ENTER` — Open the current item in a Layout
* `/` — Open search
* `?` — Open advanced filter
* `ESC` — Close current modal or drawer (one level, prompt for save as needed)
* `SHIFT-N` — Create an item (not sure about this key combo)
* `SHIFT-ESC` — Logout (not sure about this key combo)
* `META-'` — Start a comment (not sure about this key combo, liked that it's like a quote, and near the shortcut to submit the comment)
* `\` — Open Field Context Menu for currently focused field
* `OPTION-1-9` — Go to Module in order
* `[` and `]` — Previous next pages on Layout
* `.` — Toggle info sidebar open/closed
* `,` — Toggle page full-screen (not sure about key)
* `?` — Show tooltips in popover (not built yet)

**Others**
* All typical rich-tech formatting in WYSIWYG editors (including markdown)
* Confirm all form tab indexing is handled properly
* All dropdown menus items should show relevant keyboard shortcuts
* _Something_ to navigate the navigation. We'll need a soft-select state (spoiler, I hate the browser "outline" thing) to show what you are on before activating

**Review by rijkvanzanten — APPROVED** [review]

**Comment by rijkvanzanten** [comment]
@benhaynes mistakes were made. Easier to do those changes in smaller PRs, moved the list here: https://github.com/directus/directus/discussions/8407

**Pull request merged**